openmsupply/open-msupply
# 1575 logical encounter status (missed scheduled)

Fixes #1575 

# 👩🏻‍💻 What does this PR do? 
Show the logical encounter status in the `Encounter List View` and the `Encounter Detail View`. Have encased the `tags` in a light grey bubble, so that it doesn't clash with other components in the UI. Though that `Missed` and `Scheduled` weren't informative enough, so have made it `Appointment Missed` and `Appointment Scheduled`...

![Screenshot 2023-06-28 at 1 13 16 PM](https://github.com/openmsupply/open-msupply/assets/61820074/a948ca7b-30da-4edf-9b02-1937d15ac7eb)
![Screenshot 2023-06-27 at 3 11 38 PM](https://github.com/openmsupply/open-msupply/assets/61820074/bad03e03-d89b-4a64-95e6-5dcb6de9eb80)

# 🧪 How has/should this change been tested? 
- Create a patient if you do not have one already 
- Enrol the patient in a program
- Add an encounter in the past
- See `Appointment Missed`
- Add an encounter in the future
- See `Appointment Schedule`
- Encounters that have a start date of today should show nothing.

## 📃 Documentation
Document how the logical Encounter statuses work. (Missed = start date time < now, Scheduled = start date time > now)
